agent/main.go

```diff
@@ -51,9 +52,10 @@ type Information struct {
 
 func main() {
 	if os.Geteuid() != 0 {
-		logrus.Error("ShellHub must be run as root")
-		os.Exit(1)
+		logrus.Info("ShellHub run as single user mode.")
+		osauth.DefaultShadowFilename = ".shadow"
```

Comment:
What is the use case for this file?

Reply:
To explain and help users to use, I created a script shadow_helper.sh.

```
# This script is a helper to generate shadow file.
#
# shellhub agent support to define shadow path.
# The passwords of system and shellhub could be different.
#
# In single user mode,
# it allows different accounts to login but same system account.
# You can review the session log in web console.
#
#
# example:
#  $ USER=abc PASSWD=123 SHADOW_FILEANAME=.shadow ./shadow_helper.sh
#
#  $ SHADOW_FILEANAME=.shadow ./agent


```